src/common/components/Layouts/LayoutWithSearch/index.tsx

```diff
@@ -104,17 +104,25 @@ const HeaderWrapper: React.FC = () => {
 // TODO: unit test
 const LayoutWithSearch: React.FC<AllPhotoLayoutProps> = (props) => {
   const { title, children } = props;
+  const router = useRouter();
+
   return (
     <CanvasWrapper>
       <Layout>
         <Sider />
         <ContentWrapper>
           <HeaderWrapper />
-          <Content style={{ overflow: 'initial' }}>
-            <h1>{title}</h1>
-            <hr />
-            {children}
-          </Content>
+          {router.pathname === '/dashboard' ? (
+            <Content style={{ padding: '5% 0% 5% 0%', marginLeft: '5%' }}>
+              {children}
+            </Content>
+          ) : (
```

Comment:
magic logic

Reply:
maybe just add some props to allow enable/disable padding

Reply:
solve at 9e24d2430cead55c9c61f2ad9dc23aa1955a053d